__tests__/sass.test.ts

```diff
@@ -1,10 +1,99 @@
 import * as CONFIG from "./config-example";
+import { defaultSassVariableGetter } from '../src/build/internal/webpack';
 import { getGlobalFrom } from "../src/build/internal/sass";
+import sass from 'sass'
```

Comment:
> > I've marked one of the commit as 'BREAKING CHANGE' since it MAY break production code of dependent projects if they use
> > some directly some functions like 'getGlobal' programmatically.
> 
> Could you elaborate on this? In particular, what would be an example scenario when one would use `getGlobal` programmatically?

Now that you're asking, I can't think of any fairly reasonable scenario where someone would want to use it that way except for testing, which was the reason I thought it might break code (since it broke the tests code).

Reply:
I have pushed a commit that makes sure we're using `sass` consistently across the board.